### ISSUE TITLE
Dataset distribution

### ISSUE THREAD

**Issue by arpit-jadon** [lindawangg/COVID-Net#25]
There seem to be some discordance between the number of training and testing samples mentioned in the RSNA pneumonia dataset versus the dataset distribution mentioned on the github page, probably because of the multiple rows corresponding to same patient ID in the RSNA dataset’s csv file as it was supposed to be a detection task, please verify.

**Comment by PaulMcInnis** [comment]
Hello - can you please edit your issue to use the issue template? 

You are asking multiple questions at once and it is unclear what you expected vs what you saw in each case.

**Comment by arpit-jadon** [comment]
> Hello - can you please edit your issue to use the issue template?
> 
> You are asking multiple questions at once and it is unclear what you expected vs what you saw in each case.

I have shortened the issue.

**Comment by lindawangg** [comment]
This has been noted. There are no duplicates in the updated test_COVIDx.txt and the results reflect the updated no duplicate test_COVIDx.txt. For training, the models still reflect the current train_COVIDx.txt. We are working on another model, which have additional COVID samples and removed duplicate entries from the RSNA dataset.

**Comment by arpit-jadon** [comment]
OK, thanks @lindawangg . Also, the number of test samples for normal and pneumonia classes I got from the rsna data were far more than 100 (number of images per normal/pneumonia class taken in Covid-Net test set). Is it because of the too less number of covid images ?

**Comment by jdariasl** [comment]
First of all, I would like to thank you for sharing your work. The files test_COVIDx.txt included in the repository and test_split_v2 contain quite different sets of samples: The first one is distributed (100, 100 , 10) for normal, pneumonia, and COVID-19, whilst the second one is distributed (885, 1058, 10) respectively. Why did you reported the results using the smaller test set? 

When I tested the models in the second test set, the performance of both small and large models drop down for the COVID-19 class. This is what I got after evaluating the Large model in the test_split_v2 set:

[[834.  47.   4.]
 [ 97. 955.   6.]
 [  1.   0.   9.]]
Sens Normal: 0.942, Pneumonia: 0.903, COVID-19: 0.900
PPV Normal: 0.895, Pneumonia 0.953, COVID-19: 0.474
